glean-core/python/glean/metrics/ping.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, name: str, include_client_id: bool):
         """
         self._name = name
         self._handle = _ffi.lib.glean_new_ping_type(
-            _ffi.ffi_encode_string(name), include_client_id
+            _ffi.ffi_encode_string(name), include_client_id, False
```

Comment:
Also should add this as a parameter and pass along to the FFI (as for the other language bindings).